source/support/support-overview.rst

```diff
@@ -0,0 +1,179 @@
+#. Select the :guilabel:`Issues` section
+
+Use the search bar to locate an existing issue or add a new issue.
+
+.. image:: /images/subnet/issues-section.png
+   :width: 600px
+   :alt: MinIO SUBNET with the Issues section displaying a list of an organization's issues
+   :align: center
+
+Select an existing issue from the list to expand the converation or add a response.
+
+.. image:: /images/subnet/issue-expanded.png
+   :width: 600px
+   :alt: A example MinIO SUBNET issue conversation
+   :align: center
+
+Reviewing Health Data in SUBNET
+-------------------------------
+
+SUBNET provides health data about the clusters registered to the organization from the :guilabel:`Deployments` section.
+
+The view shows the total size of the org's MinIO clusters with details for each cluster.
+
+.. image:: /images/subnet/deployments-overview.png
+   :width: 600px
+   :alt: MinIO SUBNET displaying the deployments overview
+   :align: center
+
+Each of the organization's clusters display below the summary data.
+Select a deployment row to view additional health details.
+
+Deployment Health
+~~~~~~~~~~~~~~~~~
+
+The deployment's details include a summary of the deployment's configuration and the number of checks run and failed.
+You can select :guilabel:`Upload` to add diagnostic health data obtained from the :mc-cmd:`mc support diagnostics` command or the MinIO Console's Support > Health page.
+
+If you need support from MinIO Engineering, you can create a :guilabel:`New Issue` for the deployment.
+
+.. image:: /images/subnet/SUBNET-deployment-health-summary.png
+   :width: 600px
+   :alt: MinIO SUBNET displaying health summary information for a myminio deployment
+   :align: center
+
+When available, the following checks display for the deployment:
+
+- CPU Cores 
+- CPU Flags
+- CPU Match
+- Drive Match
+- Drive Status
+- Drive Usage
+- File System Match
+- Health Report
+- Memory Size
+- Internode Latency (Roundtrip)
+- MinIO Server Arguments
+- MinIO Version
+- NTP
+- Server Process
+- Swap Memory
+- System Settings
+- TLS config
+
+Failed checks display first.
+Checks with warnings display after failed checks.
+Finally, checks that pass display.
+
+Select any failed or warned checks to display the JSON output for additional details.
+You can scroll vertically through the output for the selected check.
+
+.. image:: /images/subnet/deployment-health-tls-config-fail.png
+   :width: 600px
+   :alt: MinIO SUBNET's health report for a deployment showing a failed Health Report with details expanded
+   :align: center
+
+Uploading Data to SUBNET
+------------------------
+
+If you registered the cluster with |SUBNET|, Performance and Inspection files can automatically upload to SUBNET.
+
+For clusters with an airgap, firewall, or otherwise disconnected from connecting to SUBNET directly, you can manually upload files.
+
+#. Generate the file(s) to upload from the :ref:`Console <minio-console>` or :ref:`Command Line <minio-client>`'s support functions
```

Comment:
replace <minio-client> here with `mc support diagnostic` since afaik that's the only one that supports this functionality.